libafl/src/events/launcher.rs

```diff
@@ -80,15 +80,7 @@ const LIBAFL_DEBUG_OUTPUT: &str = "LIBAFL_DEBUG_OUTPUT";
     clippy::ignored_unit_patterns
 )]
 #[derive(TypedBuilder)]
-pub struct Launcher<'a, CF, EMH, MT, S, SP>
-where
-    CF: FnOnce(Option<S>, LlmpRestartingEventManager<EMH, S, SP>, CoreId) -> Result<(), Error>,
-    EMH: EventManagerHooksTuple<S>,
-    S::Input: 'a,
-    MT: Monitor,
-    SP: ShMemProvider + 'static,
-    S: State + 'a,
-{
+pub struct Launcher<'a, CF, EMH, MT, S, SP> {
```

Comment:
Nice!

Reply:
this is a very bad pattern.
people (including me) just copy paste into it. but in fact there's 0 need to constraint in these struct definition

Reply:
True, however it's also pretty useless to create a `Launcher` where one of the generics doesn't adhere to these generics